src/app/portlets/dot-edit-content/dot-edit-content.component.spec.ts

```diff
@@ -40,6 +46,27 @@ class WorkflowServiceMock {
     }
 }
 
+class DotDevicesServiceMock {
```

Comment:
You are mocking this classes in ```dot-edit-content-view-as-toolbar.component.spec.ts``` too... we have a ```/test/``` folder where we put this mocks so we can re use them.